pages/topics/gateway-events.mdx

```diff
@@ -2159,6 +2194,151 @@ Sent when a game relationship is removed, relevant to the current user.
+
+###### Lobby Delete Structure
+
+| Field  | Type      | Description                                         |
+| ------ | --------- | --------------------------------------------------- |
+| id     | snowflake | The ID of the lobby                                 |
+| reason | string    | The [lobby deletion reason](#lobby-deletion-reason) |
+
+#### Lobby Member Add
+
+Sent when an user joins a lobby.
+
+###### Lobby Member Add Structure
+
+| Field          | Type                                                        | Description                                      |
+| -------------- | ----------------------------------------------------------- | ------------------------------------------------ |
+| member         | [lobby member](/resources/lobby#lobby-member-object) object | The added member                                 |
+| lobby_id       | snowflake                                                   | The ID of the lobby                              |
+| application_id | snowflake                                                   | The ID of the application that created the lobby |
+
+#### Lobby Member Update
+
+Sent when a lobby member is updated.
+
+###### Lobby Member Update Structure
+
+| Field          | Type                                                        | Description                                      |
+| -------------- | ----------------------------------------------------------- | ------------------------------------------------ |
+| member         | [lobby member](/resources/lobby#lobby-member-object) object | The updated member                               |
+| lobby_id       | snowflake                                                   | The ID of the lobby                              |
+| application_id | snowflake                                                   | The ID of the application that created the lobby |
+
+#### Lobby Member Remove
+
+Sent when a lobby member is removed.
+
+###### Lobby Member Remove Structure
+
+| Field          | Type                                                        | Description                                      |
+| -------------- | ----------------------------------------------------------- | ------------------------------------------------ |
+| member         | [lobby member](/resources/lobby#lobby-member-object) object | The removed member                               |
+| lobby_id       | snowflake                                                   | The ID of the lobby                              |
+| application_id | snowflake                                                   | The ID of the application that created the lobby |
+
+#### Lobby Message Create
+
+Sent when a lobby message is created. The inner payload is [message](/resources/message#message-object) object with the [extra structure](#message-object-extra-fields) above.
+
+<Alert type="warn">
+
+The message can be a partial [message](/resources/message#partial-message-object) object.
```

Comment:
```suggestion
When the lobby does not have a channel linked, the message will be a partial [message](/resources/message#partial-message-object) object.
```
not sure if this is the actual requirement but whatever it is should be mentioned